Microsoft.Toolkit.Uwp.UI.Controls/Menu/Menu.cs

```diff
@@ -12,6 +13,7 @@ namespace Microsoft.Toolkit.Uwp.UI.Controls
     /// <summary>
     /// Menu Control defines a menu of choices for users to invoke.
     /// </summary>
+    [Obsolete("This control will be removed in a future major release.")]
```

Comment:
We should add `Please use the MenuBar control from the WinUI Library instead.` to these messages.